### DISCUSSION TITLE
SSI Kit, Create Issue Verifiable Credential, Responses is not return json?

### DISCUSSION THREAD

**Discussion by tanawatu** [walt-id/.github#46] [Help] [answered]
Hello Waltid team !

Why am I getting responses in Base64 instead of JSON after following this video tutorial? I can't use the responses to check in my box. Did I do something wrong?

"video tutorial"
https://www.youtube.com/watch?v=RjM0RyUQDR0&t=204s

"this responses"  
"eyJraWQiOiJkaWQ6a2V5Ono2TWtyaXo4WEdMTkx0aTdwZDd2elZLM3FwV3dFQTNxVFlTaHFISmREd3dZd0JrRSN6Nk1rcml6OFhHTE5MdGk3cGQ3dnpWSzNxcFd3RUEzcVRZU2hxSEpkRHd3WXdCa0UiLCJ0eXAiOiJKV1QiLCJhbGciOiJFZERTQSJ9.eyJpc3MiOiJkaWQ6a2V5Ono2TWtyaXo4WEdMTkx0aTdwZDd2elZLM3FwV3dFQTNxVFlTaHFISmREd3dZd0JrRSIsInN1YiI6ImRpZDplYnNpOnpzdUo4RkVaekVjQlcyM0Q3MlNUZ3VlIiwibmJmIjoxNjgyOTI3MTM2LCJpYXQiOjE2ODI5MjcxMzYsInZjIjp7InR5cGUiOlsiVmVyaWZpYWJsZUNyZWRlbnRpYWwiLCJWZXJpZmlhYmxlQXR0ZXN0YXRpb24iLCJWZXJpZmlhYmxlVmFjY2luYXRpb25DZXJ0aWZpY2F0ZSJdLCJAY29udGV4dCI6WyJodHRwczovL3d3dy53My5vcmcvMjAxOC9jcmVkZW50aWFscy92MSJdLCJpZCI6InVybjp1dWlkOjc2YmM3NmM2LWJiNTEtNDk5OC04YWI4LTdmZTUzNzNiMmFiNyIsImlzc3VlciI6eyJpZCI6ImRpZDprZXk6ejZNa3JpejhYR0xOTHRpN3BkN3Z6VkszcXBXd0VBM3FUWVNocUhKZER3d1l3QmtFIn0sImlzc3VhbmNlRGF0ZSI6IjIwMjMtMDUtMDFUMDc6NDU6MzZaIiwiaXNzdWVkIjoiMjAyMy0wNS0wMVQwNzo0NTozNloiLCJ2YWxpZEZyb20iOiIyMDIzLTA1LTAxVDA3OjQ1OjM2WiIsImNyZWRlbnRpYWxTdWJqZWN0Ijp7ImlkIjoiZGlkOmVic2k6enN1SjhGRVp6RWNCVzIzRDcyU1RndWUiLCJmYW1pbHlOYW1lIjoiQmFuayIsImdpdmVuTmFtZXMiOiJKYW0ifX0sImp0aSI6InVybjp1dWlkOjc2YmM3NmM2LWJiNTEtNDk5OC04YWI4LTdmZTUzNzNiMmFiNyJ9.hy0zdFRcXm7-sgEsw3KUFIFbZ62HzQOllT3z7i5YolrvOTlSHjdrjJpfSu_I7FJUUk0PlRH-gEwtJPnXC8m3Cw"

i need responses like video tutorial, What should I do?
Thank you, I wish you a good day

**Comment by atharva-pb** [comment]
Hi team, I am getting the same problem (even in CLI), can some please help? Thanks

![Screenshot from 2023-05-09 00-48-57](https://user-images.githubusercontent.com/129721629/236914296-8949d055-a350-458f-a497-4b3e3178212e.png)

**Comment by taminobaumann** [comment, marked as answer]
Hey there, 

Thank you for reaching out and explaining your issue in detail. We understand that you're receiving responses in Base64 format after following the video tutorial, and this is not what you were expecting.

The response you're getting is not wrong; it's just in a different format, known as JSON Web Token (JWT). Using [jwt.io](https://jwt.io/) you can also paste the JWT and you will see the credential in the payload. It appears that the tutorial you've followed is providing responses in the JSON LD format.

To receive responses in the JSON LD format, you can adjust your configuration settings in the request body. Specifically, you should add the following line to your request body config object:
`"proofType": "LD_PROOF"`

**Example**
```
{
"templateId": "VerifiableId",
"config": {
     "issuerDid": "did:key:z6MkgjQxcoh1JWXMgG7xPNeBaqUXWu42LAW8G5KmE7J6d1CG",
     "subjectDid": "did:key:z6MkgjQxcoh1JWXMgG7xPNeBaqUXWu42LAW8G5KmE7J6d1CG",
     "proofType": "LD_PROOF"
     },
"credentialData": {
     "credentialSubject": {
         "firstName": "Tamino"
         }
     }
}
```

This should allow you to generate a credential in the JSON LD format, similar to what is shown in the video tutorial.

Please try this out and let me know how it goes. If there is anything else, please let me know.

Wishing you a great day!

Best
Tamino

**Reply by atharva-pb** [reply]
Thanks a lot Tamino!

**Reply by tanawatu** [reply]
Thanks a lot !!!!
